internal/cluster/cluster.go

```diff
@@ -1532,3 +1533,59 @@ func (m *Manager) updateSchedulableMastersForcedTrue(ctx context.Context, cluste
 
 	return nil
 }
+
+func (m *Manager) HandleVerifyVipsResponse(ctx context.Context, clusterID strfmt.UUID, stepReply string) error {
+	log := logutil.FromContext(ctx, m.log)
+	return m.db.Transaction(func(tx *gorm.DB) error {
+		var response models.VerifyVipsResponse
+		if err := json.Unmarshal([]byte(stepReply), &response); err != nil {
+			log.WithError(err).Error("HandleVerifyVipsResponse: unmarshal")
+		}
```

Comment:
I see the response is used only in line 1551, so this can be move to there.

Reply:
OK